openstack/containerinfra/v1/clustertemplates/results.go

```diff
@@ -0,0 +1,60 @@
+package clustertemplates
+
+import (
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// CreateResult is the response of a Create operations.
+type CreateResult struct {
+	commonResult
+}
+
+// Extract is a function that accepts a result and extracts a cluster-template resource.
+func (r commonResult) Extract() (*ClusterTemplate, error) {
+	var s *ClusterTemplate
+	err := r.ExtractInto(&s)
+	return s, err
+}
+
+// Represents a template for a Cluster Template
+type ClusterTemplate struct {
```

Comment:
Because of the number of fields, let's sort these in alphabetical order.

Reply:
I was following the order of the doc. Will change to alphabetical.

Reply:
Yeah, in general that would be fine. But given the amount of options here, I think deviating and going with alphabetical is a more manageable layout here.